cpp/ps0000/p0000-soln.cpp

```diff
@@ -1,9 +1,7 @@
-#include <iostream> 
-
-using namespace std;
-
-int main(void) 
+#include <iostream> // 2/2

```

Comment:
- [x] used `#` instead of `@` for directive `include`
- [x] edited header file name to only `iostream`